test/unit/contextResponses/blackButtonPollingRequestQuerySuccess.json

```diff
@@ -13,7 +13,10 @@
           },
           {
             "name": "op_result",
-            "value": "999999",
+            "value": {
+              "code": 200,
```

Comment:
We decided offline that the CA should send a string as the code and new tests cases will be included in the IOTA in a forthcoming PR... ;) So merging... :) Thanks @dmoranj !